CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Release highlights
 
+* Support for adding a `change-id` Git header behind the configuration setting `git.change-id`. If
```

Comment:
nit: If we still consider this as an experimental feature, I wouldn't add it to the release highlights. (If not, enable it by default?)

Reply:
My thought here was that it was both experimental but also something that some interested individuals would enjoy turning on and this seemed like the right place to communicate that.